src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.builds

```diff
@@ -2,21 +2,16 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="runtime.native.System.IO.Compression.pkgproj"/>
```

Comment:
These packages don't contain any content.  If they are supposed to then that's a different bug and I will add them back.